.github/file-paths.yaml

```diff
@@ -7,6 +7,7 @@ ci: &ci
 
 shared_sources: &shared_sources
   - "shared/src/**"
+  - "src/metabase/lib/**"
```

Comment:
We should figure out the way to distinguish between the source files and the tests (or other irrelevant files) in the next PR.

But let's merge this now as it is because it's more important to have this trigger than to fine tune it.

Reply:
No worries there, `src/` is all production code. The tests for Clojure files are in `test/metabase/lib`.

Reply:
Oh, right.
I confused it with the `frontend/src/metabase/lib/`